app/builders/immediate_email_builder.rb

```diff
@@ -46,6 +46,7 @@ def body(content_change, subscriptions)
 
         #{presented_unsubscribe_links(subscriptions)}
 
+        &nbsp;
```

Comment:
This is showing up in emails, which isn't exactly pleasing on the eye...

```
[Sri Lanka travel advice](https://www.gov.uk/foreign-travel-advice/sri-lanka)

 6:37am, 8 March 2018: Summary – Updated information on curfew imposed in the Kandy administrative region and related travel disruption, following a number of violent incidents – You’re
getting this email because you subscribed to Travel advice for all countries – travel advice updates on GOV.​UK.

Unsubscribe from [Travel advice for all countries – travel
advice](https://www.gov.uk/email/unsubscribe/0109de5d-5078-416b-a034-90777de8323a?title=Travel%20advice%20for%20all%20countries%20%20-%20travel%20advice)

&nbsp;

^Is this email useful? [Answer some questions to tell us more](https://www.smartsurvey.co.uk/s/govuk-email/?f=immediate).
```

Should this be an actual non breaking space, or is something going wrong with the rendering somewhere?

Reply:
That's odd – I've not seen that rendered. Can you forward me the email you got, Chris?

Reply:
The markdown is shown as in in text emails, if you view the emails as HTML they will show correctly.

Reply:
If GOV.UK Notify support horizontal rules, we might be able to use `---` instead.

Reply:
We're already using horizontal rules in the emails, though oddly can't spot that in the text version.
We tried some variations of the design using more than one horizontal rule but they all looked weird.

Reply:
We could ask Notify to support a nice way of having a large break between paragraphs.

Reply:
That would be a little more desirable. 
If viewing text only anyway, I feel that the nbsp is acceptable as a temporary visual aberration.